src/components/EarningsTable/EarningsTable.js

```diff
@@ -8,16 +8,7 @@ import {
 } from 'utilities';
 import CustomTableFilter from 'components/common/CustomTableFilter/CustomTableFilter';
 import CustomTableItemDetails from 'components/common/CustomTableItemDetails/CustomTableItemDetails';
-/**
```

Comment:
What's the reason for removing this block?

Reply:
I seems like I accidentally deleted it, I fix it now